packages/framework-core/src/booster-register-handler.ts

```diff
@@ -62,15 +58,25 @@ export class RegisterHandler {
     return boosterEventsTypesNames.includes(eventTypeName) ? BOOSTER_SUPER_KIND : DOMAIN_SUPER_KIND
   }
 
-  private static getEntityTypeName(
+  private static reducerClassName(
```

Comment:
A better naming could be `getReducerClassNameForEntity`,just leave it or take it.